lxd/instance/drivers/driver_qemu_bus.go

```diff
@@ -118,14 +118,14 @@ func (a *qemuBus) allocate(multiFunctionGroup string) (string, string, bool) {
 	if a.name == "pcie" {
 		if p.fn == 0 {
 			portName := fmt.Sprintf("%s%d", busDevicePortPrefix, a.portNum)
-			_ = qemuPCIe.Execute(a.sb, map[string]any{
-				"portName": portName,
-				"index":    a.portNum,
-				"addr":     fmt.Sprintf("%x.%d", p.bridgeDev, p.bridgeFn),
+			qemuAppendSections(a.sb, qemuPCIeSections(
+				portName,
+				a.portNum,
+				fmt.Sprintf("%x.%d", p.bridgeDev, p.bridgeFn),
 
```

Comment:
Could make this a new var to shorten line

Reply:
no longer relevant since I refactored to use struct/opts style